dhis-2/dhis-test-integration/src/test/java/org/hisp/dhis/tracker/export/event/EventChangeLogServiceTest.java

```diff
@@ -157,8 +164,6 @@ void shouldReturnChangeLogsWhenDataValueIsCreated() throws NotFoundException, Fo
   @Test
   void shouldReturnChangeLogsWhenDataValueIsDeleted()
       throws NotFoundException, IOException, ForbiddenException {
-    testAsUser("M5zQapPyTZI");
```

Comment:
Is this method still needed? should we remove it

Reply:
it is still used to inject other users in the test